stubs/seaborn/METADATA.toml

```diff
@@ -1,4 +1,4 @@
-version = "0.13.0"
+version = "0.13.*"
```

Comment:
We've had our tests broken once before by new things being added in `seaborn` micro releases ( #11212), so I worry that that could happen again. I think I'd rather just bump the micro version here? It's not a massive issue if stubsabot files frequent PRs prompting us to bump the micro version, but stubtest failing every time a seaborn micro version is release (#11212) is more of a pain -- we can't take our time over fixing it like we can when stubsabot tries to bump the version number